src/index.css

```diff
@@ -1,10 +1,31 @@
 body {
```

Comment:
All you styles should be in the App.css, can you move them over and make sure there are not duplicate styles?